deepa/Schedule/ScheduleView.swift

```diff
@@ -18,15 +19,58 @@ struct ScheduleView: View {
         _viewModel = StateObject(wrappedValue: viewModel)
         
     }
+
+    static let extractDate: DateFormatter = {
+        let formatter = DateFormatter()
+        formatter.dateFormat = "dd/MM/yyyy"
+        return formatter
+    }()
     
+    @State private var schedulePicker = 0
+
     var body: some View {
-        VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Schedule!")
+        NavigationView {
+            VStack {
+                
+                if schedulePicker == 0 {
+                    List {
+                        ForEach(viewModel.schedule) { information in
+                            if Self.extractDate.string(from: information.startAt ?? Date.now) == "20/01/2022" {
+                                VStack {
+                                    NavigationLink(destination: WorkshopDetailsView(title: information.title, startAt: information.startAt ?? Date.now, endAt: information.endAt ?? Date.now, talkDescription: information.talkDescription ?? "", activityName: information.activityName ?? "", speakers: information.speakers)) {
```

Comment:
Could you rename `information` to `talk`? structurally, a schedule is made up of talks.
then ... `WorkshopDetailsView` / `DetailView` takes the entire talk and decides how to render the talk